ui/app/src/services/users.ts

```diff
@@ -49,8 +49,9 @@ export function trash(username: string): Observable<true> {
   return del(`/studio/api/2/users?username=${encodeURIComponent(username)}`).pipe(mapTo(true));
 }
 
-export function fetchAll(options?: PaginationOptions): Observable<PagedArray<User>> {
+export function fetchAll(keyword?: string, options?: PaginationOptions): Observable<PagedArray<User>> {
```

Comment:
Add it to options argument.
Don't modify the pagination options interface just add `& { keyword?: string }`